components/cloud-foundry/src/view/applications/application/log-stream/cf-log-viewer.html

```diff
@@ -0,0 +1,27 @@
+<div class="panel panel-default summary">
+  <div class="panel-heading action-header">
+    <span class="log-stream-title" translate>Log Stream</span>
+    <span ng-if="cfLogViewer.streamingStatus !== 2 && !cfLogViewer.disableStatus" class="app-log-stream-indicator material-icons"
+    ng-class="{
+                'log-stream-ok text-primary': cfLogViewer.streamingStatus === 1,
+                'log-stream-error text-danger': cfLogViewer.streamingStatus === 3
+              }"></span>
+    <bounce-spinner classes="bounce-spinner-sm" ng-if="cfLogViewer.streamingStatus === 2 && !cfLogViewer.disableStatus"></bounce-spinner>
+    <!--// After the user has scrolled up we disable auto-scroll-->
+    <!--// They can re-enable auto-scroll either by:-->
+    <!--// - manually scrolling to the bottom-->
+    <!--// - clicking the auto-scroll button-->
+    <a class="app-log-stream-autoScroll btn btn-link" ng-click="cfLogViewer.autoScrollOn = true">
```

Comment:
This is different to the original code - we called an autoScroll() method. What's the reason for changing?